src/displayapp/widgets/Counter.cpp

```diff
@@ -0,0 +1,105 @@
+#include "displayapp/widgets/Counter.h"
+
+using namespace Pinetime::Applications::Widgets;
+
+Counter::Counter(int min, int max) : min {min}, max {max} {
+}
+
+void Counter::upBtnEventHandler(lv_obj_t* obj, lv_event_t event) {
+  auto* widget = static_cast<Counter*>(obj->user_data);
+  if (event == LV_EVENT_SHORT_CLICKED || event == LV_EVENT_LONG_PRESSED_REPEAT) {
+    widget->Increment();
+  }
+}
+
+void Counter::downBtnEventHandler(lv_obj_t* obj, lv_event_t event) {
+  auto* widget = static_cast<Counter*>(obj->user_data);
+  if (event == LV_EVENT_SHORT_CLICKED || event == LV_EVENT_LONG_PRESSED_REPEAT) {
+    widget->Decrement();
+  }
+}
+
+void Counter::Increment() {
+  value++;
+  if (value > max) {
+    value = min;
+  }
+  UpdateLabel();
+};
+
+void Counter::Decrement() {
+  value--;
+  if (value < min) {
+    value = max;
+  }
+  UpdateLabel();
+};
+
+void Counter::SetValue(int newValue) {
+  value = newValue;
+  UpdateLabel();
+}
+
+void Counter::UpdateLabel() {
+  lv_label_set_text_fmt(number, "%.2i", value);
+}
+
+void Counter::Create() {
+  counterContainer = lv_obj_create(lv_scr_act(), nullptr);
+  lv_obj_set_style_local_bg_color(counterContainer, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_MAKE(0x38, 0x38, 0x38));
```

Comment:
we could define the `LV_COLOR_MAKE(0x38, 0x38, 0x38)` only once and give it a name